homework03/arrays/arrays.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+func main() {
+    nums := []int{1, 2, 3, 4, 5, 6}
```

Comment:
It's rather a slice than an array... According to your task description you should use array. Change this please